src/plugin/vk-wrapper/src/debugMessenger/DebugMessenger.cpp

```diff
@@ -25,8 +27,16 @@ VKAPI_ATTR VkBool32 VKAPI_CALL DebugMessenger::Callback(VkDebugUtilsMessageSever
                                                         const VkDebugUtilsMessengerCallbackDataEXT *pCallbackData,
                                                         void *pUserData)
 {
-    std::cerr << "validation layer: " << pCallbackData->pMessage << std::endl;
-
+    std::unordered_map<VkDebugUtilsMessageSeverityFlagBitsEXT, ES::Plugin::Log::Level> severityMap = {
```

Comment:
Could this map be placed as a constant in the `DebugMessenger.hpp` header or out of function, above ?

Reply:
Done